tests/data/power_flow/automatic-tap-regulator/auto-tap-changer-meshed-any/input.json

```diff
@@ -0,0 +1,59 @@
+{
+  "version": "1.0",
+  "type": "input",
+  "is_batch": false,
+  "attributes": {},
+  "data": {
+    "node": [
+      {"id": 0, "u_rated": 10000},
+      {"id": 1, "u_rated": 10000},
+      {"id": 2, "u_rated": 10000},
+      {"id": 3, "u_rated": 10000},
+      {"id": 4, "u_rated": 10000},
+      {"id": 5, "u_rated": 10000},
+      {"id": 6, "u_rated": 10000},
+      {"id": 7, "u_rated": 400},
+      {"id": 8, "u_rated": 400},
+      {"id": 9, "u_rated": 400},
+      {"id": 10, "u_rated": 400}
+    ],
+    "line": [
+      {"id": 11, "from_node": 0, "to_node": 1, "from_status": 1, "to_status": 1, "r1": 0.005, "x1": 0.5, "c1": 0, "tan1": 0},
+      {"id": 12, "from_node": 0, "to_node": 2, "from_status": 1, "to_status": 1, "r1": 0.1, "x1": 0.6, "c1": 0, "tan1": 0},
+      {"id": 13, "from_node": 3, "to_node": 5, "from_status": 1, "to_status": 1, "r1": 0.005, "x1": 0.5, "c1": 0, "tan1": 0},
+      {"id": 14, "from_node": 4, "to_node": 6, "from_status": 1, "to_status": 1, "r1": 0.003, "x1": 0.3, "c1": 0, "tan1": 0},
+      {"id": 15, "from_node": 7, "to_node": 9, "from_status": 1, "to_status": 1, "r1": 0.005, "x1": 0.5, "c1": 0, "tan1": 0},
+      {"id": 16, "from_node": 8, "to_node": 9, "from_status": 1, "to_status": 1, "r1": 0.005, "x1": 0.6, "c1": 0, "tan1": 0}
```

Comment:
i chose to let the lines have various impedances to induce different optimal tap position settings